code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -160,12 +160,12 @@ Credit where due:
 		restricted_jobs += protected_jobs
 	if(CONFIG_GET(flag/protect_assistant_from_antagonist))
 		restricted_jobs += "Assistant"
-	var/starter_servants = 4 //Guaranteed four servants
+	var/starter_servants = 5 //Guaranteed five servants
 	var/number_players = num_players()
 	roundstart_player_count = number_players
-	if(number_players > 30) //plus one servant for every additional 10 players above 30
-		number_players -= 30
-		starter_servants += round(number_players / 10)
+	if(number_players > 40) //plus one servant for every additional 5 players above 40
```

Comment:
Yeah maybe revert this.
Cant wait to have 10 fucking clockies shift start because its high pop and they are now unbeatable.